.github/workflows/run-end-to-end.yml

```diff
@@ -74,13 +77,25 @@ jobs:
       with:
         name: ${{ inputs.binaries_artifact }}
         path: binaries/
+    - name: Generate Unique ID to be used for all AWS Test Resource names
+    # generates a unique ID used for Cross Tracer Propagation Tests (for naming AWS resource uniquely). We need this ID to be saved and reused for replay scenario.
+      id: generate_unique_id
+      run: |
+        HASH=$(uuidgen | tr -d '-' | head -c 10)
+        UNIQUE_ID=${HASH}_${{ inputs.library }}_${{ matrix.weblog }}
```

Comment:
I'd be more confortable with a more unique name for this en var. `SYSTEM_TESTS_INTEGRATION_UNIQUE_ID` ?

Reply:
Though, why this unique Id can't be computed inside the test process ?

Reply:
Thanks for fixing this!